src/client/js/app.js

```diff
@@ -27,19 +28,19 @@ function performAction(e) {
     };
     // Countdown
     const departureDate = document.getElementById('departure-date').value;
-    if (departureDate === '') {
-        alert('Date is not specified');
-        return;
-    }
+    // if (departureDate === '') {
+    //     alert('Date is not specified');
+    //     return;
+    // }
```

Comment:
These comments could be deleted, I don't think you need them anymore as you are switched to the HTML validation handling.